harness/setup.py

```diff
@@ -57,11 +54,6 @@
         # docker-compose has a requirement not properly propagated with semi-old pip installations;
         # so we expose that requirement here.
         "websocket-client<1",
-        # Swagger-codegen: python requirements
-        "certifi>=2017.4.17",
```

Comment:
wait, don't we still use certifi in https://github.com/determined-ai/determined/blob/d0adc51c96a778d5f786aa334f783fc79290982c/harness/determined/common/api/certs.py#L10

Reply:
Yup, I guess we do.  I think it was an unlisted dependency before.  I'll pull that back in explicitly.